src/plugins/presentation_util/public/components/dashboard_drilldown_options/types.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+export type DashboardDrilldownOptions = {
+  useCurrentFilters: boolean;
+  useCurrentDateRange: boolean;
+  openInNewTab: boolean;
+};
```

Comment:
By keeping this as a `type` rather than an `interface` (and ignoring the corresponding `consistent-type-definitions` linting error), we prevent this error in `abstract_dashboard_drilldown.tsx`:

![image](https://github.com/elastic/kibana/assets/8698078/ec35750a-4fc8-40ac-a35b-5612add59af4)